### PR TITLE
feat: documentation done with folder structure

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

File: Docs/doc.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                <span class="margin-lr" ><ion-icon name="arrow-redo-sharp"></ion-icon></span> </a>
+        </div>
+       
+    </div>
+
+</main>
+    
+</body>
+<script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+<script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
```
Comment (MaruthiPothuganti) [inline Docs/doc.html]: Script tags need to be inside body. [Check here](https://hackinbits.com/interview-questions/html/why-script-tags-should-be-placed-at-the-end-of-body-tag)

**Review by MaruthiPothuganti — APPROVED** [review]
Suggested one change. everything else looks good

File: Docs/doc.html (hunk 1 above)
Comment (govardhanreddyc79) [reply Docs/doc.html]: okay

Comment (govardhanreddyc79) [reply Docs/doc.html]: Changes done.

**Pull request merged**